packages/Webkul/Velocity/src/Resources/views/shop/products/view/cross-sells.blade.php

```diff
@@ -1,15 +1,9 @@
-@foreach ($cart->items as $item)
-    @php
-        $product = $item->product;
+@php
+$productCount = core()->getConfigData('catalog.products.product-view-page.no-of-cross-sells-product_productpage'); 
```

Comment:
correct the alignment and use snake casing in config field.

Reply:
resolved